MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -32,6 +82,26 @@ class RouteManeuverViewController: UIViewController {
         }
     }
     
+    var availableStreetLabelBounds: CGRect {
+        return CGRect(origin: .zero, size: maximumAvailableStreetLabelSize)
```

Comment:
It isn't always easy to predict when the available bounds will change. For example, the screen orientation may change, or the shield image may load asynchronously. Could we make this abbreviation mechanism run later, when we know the width of the view?

* A subclass of UILabel has a property `unabridgedText` that we set to the full, unavbreviated name.
* AbridgedLabel has a `didSet` for `bounds` that sets `text` based on the current `bounds`.